src/main/java/seedu/address/model/entry/Name.java

```diff
@@ -54,8 +54,8 @@ public String toString() {
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
-                || (other instanceof Name // instanceof handles nulls
-                        && this.fullName.equals(((Name) other).fullName)); // state check
+               || (other instanceof Name // instanceof handles nulls
```

Comment:
this line and the next should both be indented 8 spaces relative to the previous line